DiscountDiscover/View Controllers/MapViewController.h

```diff
@@ -7,11 +7,15 @@
 //
 
 #import <UIKit/UIKit.h>
+#import <GoogleMaps/GoogleMaps.h>
```

Comment:
Why did you move this dependency from the .m to the to the .h file? Ideally we only want to import dependencies in the header file when strictly necessary, but should prefer to use forward declarations when possible (using `@class` and `@protocol` instead of `#import`).